src/state/services/toaster.js

```diff
@@ -0,0 +1,9 @@
+import toast from 'react-hot-toast';
+
+export function showSuccess(msg) {
+  toast.success(msg);
```

Comment:
`toast.success` just makes me think of:
![image](https://user-images.githubusercontent.com/73324535/182434987-b3d14a2d-a235-4ff6-b4cc-75105f01facb.png)